dockerfile/anaconda-iso-creator/lorax-build

```diff
@@ -20,18 +20,18 @@ REPO_DIR=/tmp/anaconda-rpms/
 
 # create repo from provided Anaconda RPMs
 mkdir -p $REPO_DIR
-cp -a $INPUT_RPMS/* $REPO_DIR
+cp -a $INPUT_RPMS/* $REPO_DIR || echo "RPM files can't be copied!"  # We could just do the build with official repositories only
 createrepo_c $REPO_DIR
 
 # build boot.iso with our rpms
 . /etc/os-release
 # The download.fedoraproject.org automatic redirector often selects download-ib01.f.o. for GitHub's cloud, which is too unreliable; use a mirror
 # The --volid argument can cause different network interface naming: https://github.com/rhinstaller/kickstart-tests/issues/448
-lorax -p Fedora -v $VERSION_ID -r $VERSION_ID \
+lorax -p Fedora -v "$VERSION_ID" -r "$VERSION_ID" \
       --volid Fedora-S-dvd-x86_64-rawh \
       -s http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ \
       -s file://$REPO_DIR/ \
-      $@ \
-      lorax ||  cp *.log /images/
+      "$@" \
```

Comment:
This is one of the cases where I am not so sure about blindly following shellcheck. Hopefully I am wrong.